packages/ai-jsx/src/index.ts

```diff
@@ -1,48 +1,84 @@
+/**
+ * AI.JSX - The AI Application Framework for Javascript.
+ *
+ * See: https://ai-jsx.com for more details.
+ *
+ * @packageDocumentation
+ */
+
 import { v4 as uuidv4 } from 'uuid';
 import { BoundLogger, NoOpLogImplementation, LogImplementation, Logger, PinoLogger } from './core/log.js';
 
+/** @hidden */
```

Comment:
This actually is something we'd like to expose. (Sorry, I know it's a bit guess-and-check.)